ocaml/xapi/xmlm_tree.ml

```diff
@@ -0,0 +1,27 @@
+(** Conveniently output an XML tree using Xmlm *)
```

Comment:
This is fine but if we can reuse upstream code like ezxmlm I'd prefer that.

Reply:
We can reuse it after getting it into xs-opam as mentioned above. I don't think we should delay this PR because of that, unless we can just get it into xs-opam, and then release it and put it into our branch asap.